apps/cow-fi/app/(main)/careers/refer-to-earn/page.tsx

```
@@ -1,12 +1,12 @@
 'use client'
 
 import { Color } from '@cowprotocol/ui'
-
+import { initGtm } from '@cowprotocol/analytics'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)